ea_restaurant_data_loader/mix.exs

```diff
@@ -36,14 +36,15 @@ defmodule EaRestaurantDataLoader.MixProject do
       {:phoenix, "~> 1.5.3"},
       {:phoenix_ecto, "~> 4.1"},
       {:ecto_sql, "~> 3.4"},
-      {:postgrex, ">= 0.0.0"},
+      {:postgrex, ">= 0.17.3"},
       {:phoenix_live_dashboard, "~> 0.2.0"},
       {:telemetry_metrics, "~> 0.4"},
       {:telemetry_poller, "~> 0.4"},
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:joken, "~> 2.6.0"}
+      {:joken, "~> 2.6.0"},
+      {:mock, "~> 0.3.0", only: :test}
```

Comment:
Are you using this dependency? Can you remove it if not?

Reply:
Done.